src/main/java/org/folio/inventory/support/InstanceUtil.java

```diff
@@ -100,4 +99,16 @@ public static JsonObject mergeInstances(JsonObject existing, JsonObject mapped)
     mergedInstanceAsJson.put(CHILDREN_INSTANCES_PROPERTY, children);
     return mergedInstanceAsJson;
   }
+
+  /**
+   * Returns the value if s can be parsed as Integer. Returns null if s is null.
+   *
+   * @throws NumberFormatException if s is not null and cannot be parsed as Integer.
+   */
+  private static Integer asIntegerOrNull(String s) {
+    if (s == null) {
```

Comment:
Better to use StringUtils.isBlank() here

Reply:
We can rewrite method as bellow: 
return StringUtils.isBlank(s) ? null : Integer.parseInt(s)

Reply:
I think checking on "null" is enough because this field will be added automatically by trigger and we do not need several additional checks from StringUtils when we need only to check if this field is present or not.

Reply:
> We can rewrite method as bellow: return StringUtils.isBlank(s) ? null : Integer.parseInt(s)
For me is not a good decision because I need to add a comment to this check such as I added to the method.